src/commands/wp-cli.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Wp Cli
+ *
+ * @example
+ * ```
+ * cy.wpCli()
+ * ```
+ */
+export const wpCli = (command: string, ignoreFailures = false): void => {
+  const escapedCommand = command.replace(/"/g, '\\"').replace(/^wp /, '');
```

Comment:
The CodeQL alert seems to be false positive. The purpose of this is to escape double quote, not backslash.